pkg/pillar/worker/worker.go

```diff
@@ -13,9 +13,10 @@ import (
 	"github.com/lf-edge/eve/pkg/pillar/agentlog"
 )
 
-// Logger basic interface to send debug messages
+// Logger basic interface to send debug and fatal messages
 type Logger interface {
 	Debugf(format string, args ...interface{})
+	Fatalf(format string, args ...interface{})
```

Comment:
We don't use this anymore, as `worker` module is a library that will return an error and possibly log a message, but never `Fatalf()`.

Reply:
Good catch